apps/service-portal/src/auth.ts

```diff
@@ -59,6 +59,7 @@ if (userMocked) {
       ApiScope.licensesVerify,
       ApiScope.company,
       ApiScope.vehicles,
+      '@vegagerdin.is/air-discount-scheme-scope',
```

Comment:
TODO: Figure out whether scopes should be extended/augmented for privilige finetuning. As it stands this scope may be too permissive